packages/graphpack/config/webpack.config.js

```diff
@@ -47,6 +47,10 @@ module.exports = {
         test: /\.mjs$/,
         type: 'javascript/auto',
       },
+      {
+        test: /\.tsx?$/,
+        loader: 'ts-loader',
```

Comment:
I think this package should also provide a default `tsconfig.json` (and load it here from `CWD`.
I suggest to let developers create their own `tsconfig.json` inside the project repo, and not part of this package, because this way IDEs will detect it's a TypeScript project and display the build warnings.

Reply:
Yep, totally agreed! Would appreciate if you could open a PR doing this! 👍

Reply:
Tested it now, the `ts-loader` automatically detects the `tsconfig.json` from the `cwd` :)